src/services/user/services/RegistrationLink.js

```diff
@@ -0,0 +1,91 @@
+const { BadRequest, Forbidden } = require('@feathersjs/errors');
+
+const { SC_SHORT_TITLE, SC_DOMAIN } = require('../../../../config/globals');
+
+const mailContent = (firstName, lastName, registrationLink) => {
+	const mail = {
+		subject: `Einladung für die Nutzung der ${SC_SHORT_TITLE}!`,
+		content: {
+			text: `Einladung in die ${SC_SHORT_TITLE}
+    Hallo ${firstName} ${lastName}!
+    \\nDu wurdest eingeladen, der ${SC_SHORT_TITLE} beizutreten,
+    \\nbitte vervollständige deine Registrierung unter folgendem Link: ${registrationLink}
+    \\nViel Spaß und einen guten Start wünscht dir dein ${SC_SHORT_TITLE}-Team`,
+			html: '',
+		},
+	};
+	return mail;
+};
+
+const { userModel } = require('../model');
+
+const getCurrentUserInfo = (id) => userModel.findById(id)
+	.select(['schoolId', 'email', 'firstName', 'lastName'])
+	.populate('roles')
+	.lean()
+	.exec();
+
+class RegistrationLink {
+	async create(data, params) {
+		let totalMailsSend = 0;
+
+		try {
+			const { userIds } = data;
+
+			for (const userId of userIds) {
+				const account = await this.app.service('/accounts').find({
```

Comment:
why not requesting all accounts first and then run a foreach on this accounts? Will be faster and save requests to the database